packages/medusa/src/services/order.ts

```diff
@@ -1146,23 +1146,13 @@ class OrderService extends TransactionBaseService {
       const inventoryServiceTx =
         this.productVariantInventoryService_.withTransaction(manager)
 
-      const previouslyFulfilledQuantities = order.fulfillments.reduce(
-        (acc, f) => {
-          return f.items.reduce((acc, item) => {
-            acc[item.item_id] = (acc[item.item_id] || 0) + item.quantity
-            return acc
-          }, acc)
-        },
-        {}
-      )
-
       await Promise.all(
         order.items.map(async (item) => {
           if (item.variant_id) {
             return await inventoryServiceTx.deleteReservationsByLineItem(
```

Comment:
question: what does this method do? The name and call signature is a bit confusing - maybe worth renaming it. I.e.: Does it update the reservation with the quantity param - or does it reduce it with the quantity param? 

Doesn't have to be part of this PR necessarily